test/pool-cl/libraries/CLPoolParametersHelper.t.sol

```diff
@@ -11,14 +11,12 @@ contract CLPoolParametersHelperTest is Test, GasSnapshot {
         int24 tickSpacing0 = CLPoolParametersHelper.getTickSpacing(paramsWithTickSpacing0);
         assertEq(tickSpacing0, 0);
 
-        bytes32 paramsWithTickSpacingNegative13 = bytes32(uint256(0xfffff30000));
-        snapStart("CLPoolParametersHelperTest#getTickSpacing");
```

Comment:
weird but the test just failed on my machine if i keep `snapStart` & `snapEnd`

Reply:
is there a reason why we change the test case? if its just due to `snapStart() / snapEnd()` - should we only remove the `snapStart/snapEnd` and the gas snapshot file?

Reply:
oh, sorry... i didn't mean to remove this test case.. was debugging back then.. already revert it back

Reply:
gotcha, can we delete the gas snapshot file too? thanks

Reply:
i guess it's already removed 

![CleanShot 2024-06-03 at 14 39 18@2x](https://github.com/pancakeswap/pancake-v4-core/assets/137024020/0a5da906-e23d-4f1a-a411-157fc63b2b01)